Keas.Mvc/Views/Shared/_ImportEquipmentResultsScriptPartial.cshtml

```diff
@@ -0,0 +1,21 @@
+    @{await Html.RenderPartialAsync("_ImportJS");}
+    
+
+    <script type="text/javascript">
```

Comment:
@jcubbage Maybe @srkirkland disagrees, but I'd move this scrip (and the one for the key import) into the actual view, delete this partial, and just import the _ImportJS in each of the views.

Reply:
@srkirkland? I think the advantage of putting in partial is if we decide to update the versions, we can do it in one place rather then all the different calls. I did the partial inception because while import key & equipment use the same scripts, they have different calls for datatable.

Reply:
@jcubbage Keep it in the partial, just don't have a partial in a partial